functionary/prompt.py

```diff
@@ -53,36 +68,33 @@ def get_text_from_message(message: Dict) -> str:
         str: the string used in the final prompt of this message
     """
     stop_token = EndToken.from_message(message).value
-    content = message.get("content", "")
-
-    if content is not None:
-        content = f"{content}{stop_token}"
+    content = message.get("content", None)
 
     if message["role"] == "system":
-        text = "system:\n{content}\n".format(content=content)
+        text = f"system:\n{content}{stop_token}\n"
```

Comment:
can we use consistent naming. as we used EndToken, lets use end_token instead of stop_token

Reply:
sure